sql/mob_spawn_points.sql

```diff
@@ -3854,17 +3854,17 @@ INSERT INTO `mob_spawn_points` VALUES (16855054,'Offspring','Offspring',2,240.00
 INSERT INTO `mob_spawn_points` VALUES (16855055,'Offspring','Offspring',2,240.000,-60.000,0.000,190);
 
 -- You Are What You Eat
-INSERT INTO `mob_spawn_points` VALUES (16855056,'Ingester','Ingester',3,0.000,0.000,0.000,0);
+INSERT INTO `mob_spawn_points` VALUES (16855056,'Ingester','Ingester',3,-240,60,0,200);
 INSERT INTO `mob_spawn_points` VALUES (16855057,'Neoingester','Neoingester',4,0.000,0.000,0.000,0);
 INSERT INTO `mob_spawn_points` VALUES (16855058,'Neogorger','Neogorger',5,0.000,0.000,0.000,0);
 INSERT INTO `mob_spawn_points` VALUES (16855059,'Neosatiator','Neosatiator',6,0.000,0.000,0.000,0);
 INSERT INTO `mob_spawn_points` VALUES (16855060,'Wanderer','Wanderer',7,0.000,0.000,0.000,0);
-INSERT INTO `mob_spawn_points` VALUES (16855062,'Ingester','Ingester',3,0.000,0.000,0.000,0);
+INSERT INTO `mob_spawn_points` VALUES (16855062,'Ingester','Ingester',3,0.1,0,0,200);
 INSERT INTO `mob_spawn_points` VALUES (16855063,'Neoingester','Neoingester',4,0.000,0.000,0.000,0);
 INSERT INTO `mob_spawn_points` VALUES (16855064,'Neogorger','Neogorger',5,0.000,0.000,0.000,0);
 INSERT INTO `mob_spawn_points` VALUES (16855065,'Neosatiator','Neosatiator',6,0.000,0.000,0.000,0);
 INSERT INTO `mob_spawn_points` VALUES (16855066,'Wanderer','Wanderer',7,0.000,0.000,0.000,0);
-INSERT INTO `mob_spawn_points` VALUES (16855068,'Ingester','Ingester',3,0.000,0.000,0.000,0);
+INSERT INTO `mob_spawn_points` VALUES (16855068,'Ingester','Ingester',3,240,-60,0,200);
```

Comment:
Whichever of this points arent captured, but guessed, add a comment stating as much